app/lib/router/router.dart

```diff
@@ -224,7 +223,7 @@ List<RouteBase> makeRoutes(Ref ref) => [
               );
             },
             child: EditSpacePage(
-              space: state.extra as Space,
+              spaceId: state.extra as String,
```

Comment:
same here... why not:
`spaceId: state.queryParameters['spaceId'],` ?

Reply:
I will change to ```queryParameters```.